src/main/java/io/cdap/delta/bigquery/BigQueryEventConsumer.java

```diff
@@ -557,6 +559,7 @@ public synchronized void applyDML(Sequenced<DMLEvent> sequencedEvent) throws Exc
 
     latestOffset = event.getOffset();
     latestSequenceNum = sequenceNumber;
+    LOG.debug("DML id:{} offset: {} seq num : {}", event.getRow().get("id"), latestOffset.get(), latestSequenceNum);
```

Comment:
nit: remove these debug logs.

Reply:
as discussed offline, change it to trace level.